tests/test_backends/test_httomolibgpu.py

```diff
@@ -154,9 +154,9 @@ def test_remove_outlier_memoryhook(flats, ensure_clean_memory, dtype, slices):
 
 
 @pytest.mark.cupy
-@pytest.mark.parametrize("slices", [64, 128])
-@pytest.mark.parametrize("dim_x", [81, 260, 320])
-@pytest.mark.parametrize("dim_y", [340, 135, 96])
+@pytest.mark.parametrize("slices", [2, 4, 8])
+@pytest.mark.parametrize("dim_x", [1024, 2048])
+@pytest.mark.parametrize("dim_y", [1024, 2048])
```

Comment:
Why have these been changed?

Reply:
Yep, ok, that's reasonable.

However, I'm wondering if that should be done in the PR that fixes the paganin memory hooks rather than here? Is there a specific reason to put it in this PR (given that the PR is titled "CI issue fixing + tests", while this change doesn't seem to do any CI fixing or test fixing I think)?

Reply:
sure, I added it back with [291739a](https://github.com/DiamondLightSource/httomo/pull/425/commits/291739addf1e0eff4aaad54ecab902674cd60f38)